### PR TITLE
✨ Enable Likecoin transation page to redirect to ISCN transaction page and vice versa

### PR DESCRIPTION
**Pull request by shellyYG**

### CODE REVIEW

File: pages/in/tx/_id/index.vue [hunk 1]
```diff
@@ -345,6 +345,9 @@ export default {
         tx = await getCosmosTransferInfo(this.txId, {
           blocking: true,
         });
+        if (tx.isISCNTx) {
+          this.$router.replace(`/in/tx/iscn/${this.txId}`);
```
Comment (williamchong) [inline pages/in/tx/_id/index.vue:349]: Should `return` here and stop the logic below

File: pages/in/tx/iscn/_id/index.vue [hunk 2]
```diff
@@ -423,6 +423,9 @@ export default {
       const tx = await getISCNTransferInfo(this.txId, {
         blocking: true,
       });
+      if (tx.isLIKETx) {
+        this.$router.replace(`/in/tx/${this.txId}`);
```
Comment (williamchong) [inline pages/in/tx/iscn/_id/index.vue:427]: Should `return` here and stop the logic below

File: util/CosmosHelper.js [hunk 3]
```diff
@@ -151,6 +151,11 @@ export async function getTransferInfo(txHash, opt) {
         to = outputs.length > 1 ? outputs.map(o => o.address) : outputs[0].address;
         amounts = outputs.length > 1 ? outputs.map(o => o.coins[0]) : [outputs[0].coins[0]];
       }
+    } else if (typeUrl === '/likechain.iscn.MsgCreateIscnRecord'
+              || typeUrl === '/likechain.iscn.MsgUpdateIscnRecord'
+              || typeUrl === '/likechain.iscn.MsgChangeIscnRecordOwnership'
```
Comment (williamchong) [inline util/CosmosHelper.js:156]: `typeUrl.includes('/likechain.iscn')` would be simpler

File: util/cosmos/iscn/query.js [hunk 4]
```diff
@@ -121,10 +125,15 @@ export async function getISCNTransferInfo(txHash, opt) {
 
   const block = await apiClient.getBlock(height);
   const { header: { time: timestamp } } = block;
-
+  let isLIKETx = false;
```
Comment (williamchong) [inline util/cosmos/iscn/query.js:128]: Naming as `isLIKETransferTx` seems clearer

File: util/cosmos/iscn/query.js [hunk 5]
```diff
@@ -145,9 +154,33 @@ export async function getISCNTransferInfo(txHash, opt) {
...
+    } else if (bodyMessages.length === 1) {
+      const { typeUrl } = bodyMessages[0];
+      if (typeUrl === '/cosmos.bank.v1beta1.MsgSend' || typeUrl === '/cosmos.bank.v1beta1.MsgMultiSend') {
+        isLIKETx = true;
+        return { isLIKETx };
+      }
+      return new Error('Not an known transaction type');
+    } else {
+      return new Error('Not an known transaction type');
+    }
```
Comment (williamchong) [inline util/cosmos/iscn/query.js:183]: Why do we need this part if we already check typeUrl in line 131?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: util/cosmos/iscn/query.js (hunk 5 above)
Comment (shellyYG) [reply util/cosmos/iscn/query.js:183]: because line 131 actually did not have any effect....due to parsed.tx.body.messages = []
Will revise the code!

**Pull request merged**